### PR TITLE
Functions for SOU: L2

### PR DESCRIPTION
**Pull request by atsuki-kuwata**
I added functions for multiband map, by borrowing from sot by @HajimeKawahara. 

`jaxsot.core.map gen_multibandmap()` <- `sot.core.toymap make_maltiband_map()`
`jaxsot.io.reflectivity get_meanalbedo()` <- `sot.core.io_refdata get_meanalbedo()`
`jaxsot.io.reflectivity set_meanalbedo(), plot_albedo()` <- `sot.core.io_surface_type set_meanalbedo(), plot_albedo()`

### CODE REVIEW

File: src/jaxsot/core/map.py [hunk 1]
```diff
@@ -0,0 +1,34 @@
+import numpy as np
+from jaxsot.io.reflectivity import set_meanalbedo
+
+def gen_multibandmap(cmap, refsurfaces, sky, vals, bands, onsky=False):
+    """multiband map generator
```
Comment (HajimeKawahara) [inline src/jaxsot/core/map.py:5]: Can you give Args and Returns in Napoleon form?

Comment (HajimeKawahara) [reply src/jaxsot/core/map.py:5]: Like

    """compute Omega vertor of a Healpix sphere
    Args:
       nside: Nside of Healpix
    Returns:
       Omega vector (ndarray)
    """

**Comment by atsuki-kuwata** [comment]
Thank you. I have updated Args and Returns.

File: src/jaxsot/core/lc.py [hunk 2]
```diff
@@ -4,10 +4,18 @@
...
+       sigma_relative: standard deviation of noise
+
+    Returns:
+       light curve
+
     """
     lc=np.dot(W,mmap)
     sigma=np.mean(lc)*sigma_relative
-    noise=np.random.normal(0.0,sigma,len(lc))
+    noise=np.random.normal(0.0,sigma,np.shape(lc))
```
Comment (atsuki-kuwata) [inline src/jaxsot/core/lc.py:18]: modified for multiband map

**Comment by HajimeKawahara** [comment]
thanks. confirmed

**Pull request merged**